### PR TITLE
Notifications and Events base PR

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

#3737

This PR will provide the base tugboat environment and branch for the notifications/events.  This is necessary because the danse module requires PHP 7.2.  Once #3775 or #1004 is complete then this can be merged to master.

New PRs for events/notifications should be made to this branch.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

---
This change is [Reviewable](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/3996)

### CODE REVIEW

**Review by acrollet — APPROVED** [review]
Code changes look good and has been running well for me locally.

**Comment by indytechcook** [comment]
Now that php 7.2 has been merged this PR is unblocked.

File: .tugboat/config.yml [hunk 1]
```diff
@@ -42,6 +42,8 @@ services:
         # Set unlimited memory limit for PHP CLI to allow Composer to run.
         - echo "memory_limit = -1" >> /usr/local/etc/php/conf.d/php-cli.ini
         - echo "max_execution_time = 1800" >> /usr/local/etc/php/conf.d/my-php.ini
+        # Set session timeout
+        - echo "session.cookie_lifetime = 0" >> /usr/local/etc/php/conf.d/my-php.ini
```
Comment (indytechcook) [inline .tugboat/config.yml:46]: this can be removed

File: .lando.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: va-gov-cms
 recipe: drupal8
 config:
   webroot: docroot
-  php: "7.2.32"
+  php: "7.2"
```
Comment (indytechcook) [inline .lando.yml:5]: This should probably be removed also?

File: .tugboat/config.yml [hunk 3]
```diff
@@ -71,6 +73,9 @@ services:
         - ln -snf "${TUGBOAT_ROOT}/config" "${DOCROOT}/../config"
         - ln -snf "${TUGBOAT_ROOT}/files-private" "${DOCROOT}/../files-private"
 
+        # composer 1 :(
```
Comment (indytechcook) [inline .tugboat/config.yml:76]: Remove this also.

**Comment by ndouglas** [comment]
I'm gonna look into getting the Behat tests to pass... haven't dug in deep but look like spec test failutres ATM.

**Review by ndouglas — APPROVED** [review]
@indytechcook I'm gonna merge this (with the Lando PHP version changed from "7.2.32" to "7.2") unless you have any concerns.

**Pull request merged**